docs/source/guides/core_debug.rst

```diff
@@ -0,0 +1,42 @@
+##################################
+Compile Arduino libs with ESP_LOGx
+##################################
+
+There are 2 primary approaches and both of them involve editing file ``configs/defconfig.common``.
+Edit the file directly and then build.
+Later you can ``git restore configs/defconfig.common`` to go back.
+Copy the file ``cp configs/defconfig.common configs/defconfig.debug`` and edit the debug version.
+
+``vim configs/defconfig.common`` or ``vim configs/defconfig.debug``
+
+Edit **line 44** containing by default ``CONFIG_LOG_DEFAULT_LEVEL_ERROR=y`` to one of the following lines depending on your desired log level:
+
+.. code-block:: bash
+
+   CONFIG_LOG_DEFAULT_LEVEL_NONE=y # No output
+   CONFIG_LOG_DEFAULT_LEVEL_ERROR=y # Errors - default
+   CONFIG_LOG_DEFAULT_LEVEL_WARN=y # Warnings
+   CONFIG_LOG_DEFAULT_LEVEL_INFO=y # Info
+   CONFIG_LOG_DEFAULT_LEVEL_DEBUG=y # Debug
+   CONFIG_LOG_DEFAULT_LEVEL_VERBOSE=y # Verbose
+
+Then simply build the libs for all SoCs or one specific SoC. Note that building for all SoCs takes a lot of time, so if you are working only with specific SoC(s), build only for those.
+
+
```

Comment:
Here are two empty lines probably by typo

Reply:
@PilnyTomas could you fix this please?

Reply:
done in [cb3ac63](https://github.com/espressif/arduino-esp32/pull/7193/commits/cb3ac630c294eabddf9adea84ba5a4ecfe2c02c7)